edgelet/iotedged/src/app.rs

```diff
@@ -54,8 +60,7 @@ fn init_common(running_as_windows_service: bool) -> Result<Settings, Error> {
         }
     }
 
-    info!("Starting Azure IoT Edge Security Daemon");
-    info!("Version - {}", edgelet_core::version_with_source_version());
+    start_message();
```

Comment:
Just do `if cfg!()` instead of a new function. As a bonus you wouldn't need to duplicate the `info!("Version")` line.